src/main/java/com/synopsys/integration/blackduck/codelocation/signaturescanner/command/ScanCommand.java

```diff
@@ -52,6 +55,10 @@ public class ScanCommand {
     private final String correlationId;
     private final String bomCompareMode;
 
+    private List<String> command;
+
+    private Map<String, List<Integer>> commandKeysToKeyRelatedIndeces;
```

Comment:
Possible typo here with `indeces`? Is that meant to be `indices`? I also see some variables names include `indexes` (example: overriddenArgIndexes) in this file so it might be easier for future maintenance to use the same term if it all means the same?